site/src/modules/resources/AgentDevcontainerCard.tsx

```diff
@@ -0,0 +1,73 @@
+import Link from "@mui/material/Link";
+import type { Workspace, WorkspaceAgentDevcontainer } from "api/typesGenerated";
+import { ExternalLinkIcon } from "lucide-react";
+import type { FC } from "react";
+import { portForwardURL } from "utils/portForward";
+import { AgentButton } from "./AgentButton";
+import { AgentDevcontainerSSHButton } from "./SSHButton/SSHButton";
+import { TerminalLink } from "./TerminalLink/TerminalLink";
+
+type AgentDevcontainerCardProps = {
+	container: WorkspaceAgentDevcontainer;
+	workspace: Workspace;
+	host: string;
+	agentName: string;
+};
+
+export const AgentDevcontainerCard: FC<AgentDevcontainerCardProps> = ({
+	container,
+	workspace,
+	agentName,
+	host,
+}) => {
+	return (
+		<section
+			className="border border-border border-dashed rounded p-6 "
+			key={container.id}
+		>
+			<header className="flex justify-between">
+				<h3 className="m-0 text-xs font-medium text-content-secondary">
+					{container.name}
+				</h3>
+
+				<AgentDevcontainerSSHButton
+					workspace={workspace.name}
+					container={container.name}
+				/>
+			</header>
+
+			<h4 className="m-0 text-xl font-semibold">Forwarded ports</h4>
```

Comment:
What do you think about removing this?

Reply:
I personally like that. I would think on removing it later on when we have more things to display on the screen.

Reply:
Maybe it could be a little smaller? This is just a nit though, and I'm totally fine with leaving it as-is.